lib/common/prom.utils.ts

```diff
@@ -91,13 +91,6 @@ export const findOrCreateMetric = ({
 
   }
 
-  if (metric instanceof client.Counter === false) {
-    return new client.Counter({
-      name: getMetricToken(type, name),
-      help: help || `${name} ${type}`,
-    });
-  }
-
```

Comment:
Not sure about this snippet (i can be wrong). I get 

`A metric with the name *Gauge has already been registered.` 

when trying to use same gauge in code and i can't determine reason to keep this "if". Mb you can explain it to me.

Reply:
I don't really remember why I've done that.